src/aleph_client/synchronous.py

```diff
@@ -78,6 +78,12 @@ def create_program(
     persistent: bool = False,
     encoding: Encoding = Encoding.zip,
     volumes: Optional[List[Dict]] = None,
+    persistent_volumes:Optional[Tuple[str, str, str]] =None,
+    ephemeral_volumes:Optional[List[str]] =None,
+    immutable_volumes: Optional[Tuple[str, bool]] =None,
```

Comment:
These three volume types should be all included in volumes. They share the same parent class.